cmd/keeper/cmd/keeper.go

```diff
@@ -57,7 +57,10 @@ var CmdKeeper = &cobra.Command{
 	Version: cmd.Version,
 }
 
-const maxPostgresTimelinesHistory = 2
+const (
+	maxPostgresTimelinesHistory = 2
+	defaultWalKeepSegments      = 8
```

Comment:
@hmac had a suggestion that this should maybe be `minWalKeepSegments` now?

Reply:
@lawrencejones Updated.